index.html

```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
```

Comment:
```suggestion
  </TR>
```
making a minor change to illustrate the power of "Github Suggestion".

Reply:
done